packages/translations/package.json

```
@@ -54,13 +54,13 @@
     "i18next-multiload-backend-adapter": "^1.0.0"
   },
   "peerDependencies": {
-    "i18next": "^20.0.0",
+    "i18next": "^21.0.0",
```

this also requires giving the translations package a major bump, so we don't slip in a breaking change for users down stream

---

```suggestion
    "i18next": "20 - 21",
```
is the best solution here I think. we've also used this in the eslint config: https://github.com/bothrs/eslint-config/blob/main/package.json#L83
notice also that peer dependencies never have a `^` or full version number like `21.0.0`